internal/consensus/state_test.go

```diff
@@ -2215,7 +2215,7 @@ func TestPrepareProposalReceivesVoteExtensions(t *testing.T) {
 	addr := pv1.Address()
 	voteCh := subscribeToVoter(ctx, t, cs1, addr)
 
-	// ensure that the height is commited.
+	// ensure that the height is committed.
```

Comment:
:facepalm:

Reply:
I would've left it, but the linter made me fix it :smile: